api/lib/domain/models/CertificationContract.js

```diff
@@ -5,7 +5,15 @@ class CertificationContract {
 
   /* PUBLIC INTERFACE */
   static assertThatWeHaveEnoughAnswers(listAnswers, listChallenges) {
-    if (listAnswers.length < listChallenges.length) {
+
+    const someUnansweredChallenges = _.some(listChallenges,
+      (challenge) => {
+        return !listAnswers.find((answer) => answer.challengeId === challenge.challengeId)
+          && !challenge.hasBeenSkippedAutomatically
+          && !challenge.isNeutralized;
```

Comment:
Pour optimiser on boucle sur la liste en dernier (le find ne sera pas executé si l'un des 2 premiers echoue)
```suggestion
        return !challenge.hasBeenSkippedAutomatically
          && !challenge.isNeutralized;
         &&  !listAnswers.find((answer) => answer.challengeId === challenge.challengeId)

```